frontend/src/integrations/github/components/github-settings-bulk-select.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <app-dialog v-model="isModalVisible" title="Bulk selection">
+    <template #content>
+      <div class="px-6">
+        <article class="pb-3">
+          <p class="text-sm font-medium text-black mb-1">
+            Repositories
+          </p>
+          <el-select
+            v-model="form.repositories"
+            placeholder="Select option(s)"
+            class="w-full"
+            placement="bottom-end"
+            filterable
+            multiple
+            :teleported="false"
+          >
+            <el-option
+              v-for="repo of props.repositories"
```

Comment:
Can you add a first option where the user can select "All repositories". I mentioned in this comment https://linear.app/lfx/issue/LFX-1000/map-repositories-to-projects-in-chunkspages-when-enabling-github#comment-7a390807